internal/core/command/get.go

```diff
@@ -49,5 +63,11 @@ func NewGetCommand(
 		request.Header.Set(clients.CorrelationHeader, correlationID.(string))
 	}
 
+	addHeadersToRequest(request, r.Header.Get(clients.ContentType))
+
 	return newServiceCommand(device, httpCaller, request, lc), nil
 }
+
+func addHeadersToRequest(r *http.Request, value string) {
```

Comment:
Could be move this function to a more common place? This is being used by `put.go` as well and may better serve in a common location.